### PR TITLE
update image tag for nasa-singleuser image

### PR DESCRIPTION
**Pull request by batpad**
Refs https://github.com/NASA-IMPACT/veda-jh-environments/issues/46

Updates the nasa-veda-singleuser image to an image that uses an updated version of the `pangeo-notebook` base image (now using `pangeo/pangeo-notebook:2024.03.13`)

This update is needed to address https://github.com/jupyterhub/jupyter-server-proxy/security/advisories/GHSA-w3vc-fx9p-wp4v

cc @yuvipanda @ranchodeluxe @wildintellect @abarciauskas-bgse

### CODE REVIEW

File: config/clusters/nasa-veda/common.values.yaml
Comment (yuvipanda) [inline config/clusters/nasa-veda/common.values.yaml:71]: You should update the config for the GHG and ESDIS hubs as well here.

**Review by yuvipanda — CHANGES_REQUESTED** [review]
Thanks for opening the pR!

File: config/clusters/nasa-veda/common.values.yaml
Comment (slesaad) [reply config/clusters/nasa-veda/common.values.yaml:71]: let's wait for the esdis hub until the end of the webinar tomorrow Mar 21st at 1 pm CST

Comment (batpad) [reply config/clusters/nasa-veda/common.values.yaml:71]: @yuvipanda updated the PR to include the GHG and ESDIS hubs. Let's wait for a review from @ranchodeluxe as well as Mar 21st 1pm CST as per @slesaad's comment :)

Comment (slesaad) [reply config/clusters/nasa-veda/common.values.yaml:71]: we're good to go with this!

File: config/clusters/nasa-ghg/common.values.yaml
Comment (wildintellect) [inline config/clusters/nasa-ghg/common.values.yaml:69]: is it supposed to be 2024-03-20?

**Review by wildintellect — COMMENTED** [review]
2 say 2023-03-20 and one says 2024-03-20 <- please check and fix

**Comment by batpad** [comment]
@wildintellect oops! Thank you so so much for the catch there 🙏 - fixed the two wrong dates on the tags and pushed.

@wildintellect I did a very basic test of the image working on the staging VEDA hub - do you know if there's a process / more thorough tests we should be running before merging?

File: config/clusters/nasa-ghg/common.values.yaml
Comment (yuvipanda) [reply config/clusters/nasa-ghg/common.values.yaml:69]: This probably also needs to be updated, @batpad

**Review by yuvipanda — CHANGES_REQUESTED** [review]

**Comment by batpad** [comment]
Oops, must remember to actually `git add` and `git push` after making a change :-/ - thanks for the catch @yuvipanda and apologies all for all the typos. This should be actually done now.

**Review by consideRatio — APPROVED** [review]

**Comment by wildintellect** [comment]
@batpad we do not have a testing framework at the moment but it's in the works, don't let it hold up this PR. @jsignell can comment if there's anything we can do now.

Looks like you now need to:
- [ ] update from `master`
- [ ] request re-review from @yuvipanda

**Comment by sgibson91** [comment]
I actually think we can just merge this - I can override the change requested blockage and Erik has already approved

**Pull request merged**
